unasyncd/transformers.py

```diff
@@ -179,19 +180,18 @@ def __init__(
         self._post_transforms: list[Callable[[str, str, cst.Module], str]] = []
         self._file_name = file_name
         if ruff_fix:
-            self._post_transforms.append(self._run_ruff)
-
-    def _run_ruff(self, source: str, output: str, tree: cst.Module) -> str:
-        args = [
-            sys.executable,
-            "-m",
-            "ruff",
-            "check",
-            "--no-cache",
-            "--fix",
-            "--fix-only",
-            "--quiet",
-        ]
+            self._post_transforms.append(self._ruff_fix)
+        if ruff_format:
+            self._post_transforms.append(self._ruff_format)
+
+    def _run_ruff(self, source: str, output: str, tree: cst.Module, mode: str) -> str:
```

Comment:
Is this approach alright @provinzkraut?

Reply:
I'm not a huge fan of passing flags like this. I've left a different suggestion!

Reply:
Thank you!